sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2393,6 +2395,40 @@ def expand(self, pcoll):
           'The method to read from BigQuery must be either EXPORT'
           'or DIRECT_READ.')
 
+  def produce_pcoll_with_schema(self, project_id, dataset_id, table_id):
```

Comment:
Since this doesn't actually use `self`, I think it would be a good idea to pull out this logic into a helper function. [bigquery_tools.py](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/gcp/bigquery_tools.py) could be a good place, or we might define a new file for it, like `bigquery_schema_tools.py` (it looks like there's precedent for this, see `bigquery_avro_tools.py`). Would you have a preference @pabloem?